From 5ef3b4abce82a56fbbfe99ee5ba112f64eef596a Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Mon, 6 Jun 2022 23:16:44 +0200
Subject: [PATCH] arm64: dts: qcom: ipq6018: add MP5496 LDO2 node

Add support for the MP5496 LDO2 which is the SDHCI VQMMC supply.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 arch/arm64/boot/dts/qcom/ipq6018.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm64/boot/dts/qcom/ipq6018.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq6018.dtsi
@@ -817,6 +817,12 @@
 					regulator-max-microvolt = <1062500>;
 					regulator-always-on;
 				};
+
+				ipq6018_l2: l2 {
+					regulator-min-microvolt = <1800000>;
+					regulator-max-microvolt = <3300000>;
+					regulator-always-on;
+				};
 			};
 		};
 	};
